src/RtAudioInterface.cpp

```diff
@@ -417,47 +456,31 @@ int RtAudioInterface::stopProcess()
 
 //*******************************************************************************
 void RtAudioInterface::getDeviceList(QStringList* list, QStringList* categories,
-                                     bool isInput)
+                                     QList<int>* channels, bool isInput)
 {
     RtAudio baseRtAudio;
```

Comment:
I think there's a member mRtAudio for this.

Reply:
Definitely some work to be done in this area. There might be ways to refactor this so it runs faster, particularly on Windows. This PR is getting a bit thick so maybe I'll hold off on this for another time.